src/components/CToolTip/index.tsx

```diff
@@ -0,0 +1,68 @@
+import styled from 'styled-components';
+import Tippy from '@tippyjs/react';
+import { roundArrow } from 'tippy.js';
+
+interface ToolTipProps {
+  title: string;
+  text: string;
+  children: JSX.Element | React.ReactNode;
+  visible: boolean;
+  setVisible: (_: boolean) => void;
+  placement: 'top' | 'bottom';
+}
+
+const CToolTip = ({ text, title, placement, children, visible, setVisible }: ToolTipProps) => {
+  const show = () => setVisible(true);
+  const hide = () => setVisible(false);
+
+  const ArrowStyle = styled.div`
+    .tippy-svg-arrow {
+      transform: ${placement === 'bottom' ? 'none' : 'rotateX(180deg)'} !important;
+      width: 100%;
+      display: flex;
+      justify-content: ${placement === 'bottom' ? 'end' : 'center'};
+      fill: #050142;
+      margin-top: -1px;
+      margin-left: ${placement === 'bottom' ? '-8px' : '3px'} !important;
+      top: ${placement === 'bottom' ? '-5px' : 'none'};
+    }
+
+    .tippy-box {
+      line-height: 0.9em;
+    }
+  `;
```

Comment:
ArrowStyle should be outside of this component. In a separate file in a same directory as this file.